### PR TITLE
request views: reroute request views to general view

### PR DESCRIPTION
**Pull request by NRodriguezcuellar**
:heart: Thank you for your contribution!

closes https://github.com/inveniosoftware/invenio-requests/issues/186
closes https://github.com/inveniosoftware/invenio-requests/issues/184

### CODE REVIEW

File: invenio_app_rdm/requests_ui/views/requests.py [hunk 1]
```diff
@@ -78,11 +75,55 @@ def requests_detail(request=None, pid_value=None):
...
-    except TemplateNotFound:
-        return render_template(
-            "invenio_requests/details/index.html",
-            # TODO: implement and use request UI serializer
-            request=request_dict,
-            record=topic_record,
-            default_query_config=default_query_config
-        )
+
+def invitation_details(community_pid_value=None,
```
Comment (zzacharo) [inline invenio_app_rdm/requests_ui/views/requests.py:79]: Shouldn't this require the user to login?

Comment (NRodriguezcuellar) [reply invenio_app_rdm/requests_ui/views/requests.py:79]: it still has the `@login_required decorator` :smile:

Comment (zzacharo) [reply invenio_app_rdm/requests_ui/views/requests.py:79]: I see it in the `requests_detail` view but not on this one, no? If so, maybe you can move it to the `requests_view` one

Comment (NRodriguezcuellar) [reply invenio_app_rdm/requests_ui/views/requests.py:79]: ah my bad, you are right.

**Review by zzacharo — APPROVED** [review]

**Pull request merged**